examples/mpi/candle-nvidia/Dockerfile

```diff
@@ -0,0 +1,138 @@
+# ch-test-scope: full
+FROM nvcr.io/hpc/candle:20180326
```

Comment:
This raises some interesting questions:

1. Do we want an example in open source Charliecloud that requires login somewhere? On the one hand, nVidia is a big player here, it would be useful to demonstrate something with `docker login`, and it would be useful to demonstrate a vendor-provided image. On the other hand, it seems awkward to require a third-party account to run the complete test suite.
2. Are we allowed to distribute the results? Currently the results of `make build` (i.e., `$CH_TEST_TARDIR`) can be given to anyone. If we add this authenticated base image, does that change?